### PR TITLE
G UI/Pack/3

### PR DESCRIPTION
**Pull request by GuyomT**
Implementation of Unpack

### CODE REVIEW

**Review by pariciadra — APPROVED** [review]
I'm not sure that the bin is usefull in push, LGTM otherwise

**Review by LucasTesnier — APPROVED** [review]
NTR!

File: src/zappy_gui_src/Core/Network/Unpack.hpp [hunk 1]
```diff
@@ -30,8 +30,99 @@ std::map<std::string, player_orientation_e> mapOrientation{
     {"3", WEST},
 };
 
+class Player {
```
Comment (osvegn) [inline src/zappy_gui_src/Core/Network/Unpack.hpp:33]: move variables in `private` class part. They don't be used as `public`. If you need them, make setter and getter function for these variables.

File: src/zappy_gui_src/Core/Network/Unpack.hpp [hunk 2]
```diff
@@ -42,18 +133,32 @@ class Unpack {
 
     /// \brief Unpack an inventory
     /// \param inv the string of inventory to Unpack
-    void UnpackInventory(std::string &inv);
+    std::vector<int> UnpackInventory(std::string &inv);
 
     /// \brief Unpack an player
-    /// \param unpacked the vector of strings to Unpack strings
-    void UnpackPlayer(std::vector<std::string> &unpacked);
+    //i/ \param unpacked the vector of strings to Unpack strings
```
Comment (osvegn) [inline src/zappy_gui_src/Core/Network/Unpack.hpp:139]: remove the useless `i`

**Review by osvegn — CHANGES_REQUESTED** [review]
Minor changes to do, but look good

**Pull request merged**